src/renderer.ts

```diff
@@ -20,25 +15,22 @@ export interface ICoverageLines {
 
 export class Renderer {
     private configStore: IConfigStore;
-    private outputChannel: OutputChannel;
-    private eventReporter: Reporter;
+    private topSectionFinder: TopSectionFinder;
 
     constructor(
         configStore: IConfigStore,
-        outputChannel: OutputChannel,
-        eventReporter: Reporter,
+        topSectionFinder: TopSectionFinder,
     ) {
         this.configStore = configStore;
-        this.outputChannel = outputChannel;
-        this.eventReporter = eventReporter;
+        this.topSectionFinder = topSectionFinder;
     }
 
     /**
      * Renders coverage to editors
      * @param sections cached set of sections
      * @param textEditors currently visible text editors
      */
-    public async renderCoverage(
+    public renderCoverage(
```

Comment:
@ryanluker - I don't think this function needs to be `async`...

Reply:
yep doesnt look that way good catch!